src/main/java/com/workspace/io/AsyncWriter.java

```diff
@@ -0,0 +1,93 @@
+package com.workspace.io;
+import com.rabbitmq.client.ConnectionFactory;
+import com.rabbitmq.client.Connection;
+import com.rabbitmq.client.Channel;
+import java.io.IOException;
+
+public class AsyncWriter {
+
+    private static String uri=null;
+    private static Connection connection = null;
+    private static Channel channel=null;
+    private static ConnectionFactory factory = null;
+    private static AsyncWriter asyncWriter = null;
+    void init() throws Exception{
+            uri = System.getenv("RMQ_URL");
+        factory = new ConnectionFactory();
+        factory.setUri(uri);
+        factory.setConnectionTimeout(30000);
+        factory.useSslProtocol("TLSv1.2");
+        try{
+        connection = factory.newConnection();
+        }catch (Exception e){
+            System.out.println("CONN ERR: "+ e.fillInStackTrace());
+        }
+        channel = connection.createChannel();
+        try {
+            declareQueue();
+        }catch (Exception e){
+            throw e;
+        }
+
+    }
+    void declareQueue() throws IOException {
+        channel.queueDeclare("msjq",true,false,false,null);
+    }
+    public void write(String message){
+        try {
+            init();
+        }catch (Exception e){
+            System.out.println("RMQ ERROR: " + e.getMessage());
+        }
+        try{
```

Comment:
Instead of multiple try and catch block, you can have 1 try block and multiple catch blocks.